feat(nextjs): Update Next.js docs for v8 (#9910)
getsentry/sentry-docs: platform-includes/getting-started-primer/javascript.nextjs.mdx

- [Breadcrumbs](/platforms/javascript/enriching-events/breadcrumbs/) created for outgoing HTTP request with XHR and Fetch, and console logs
- [Release health](/product/releases/health/) for tracking crash-free users and sessions
- Automatic [Performance Monitoring](/product/performance/) for both the client and server
- Errors and Performance monitoring support for [Middleware](https://vercel.com/docs/concepts/functions/edge-middleware) and [Edge routes](https://vercel.com/docs/concepts/functions/edge-functions) in Vercel's edge runtime

<Note>

> `next dev --turbo` is not supported while Turbopack is in beta. Running your devserver with Turbopack will cause the Sentry Next.js SDK not to be loaded in the browser and data of serverside instrumentation will be inaccurate or incomplete.

Reviewer: ```suggestion
`next dev --turbo` is not supported while Turbopack is in beta. Running your devserver with Turbopack will cause the Sentry Next.js SDK not to be loaded in the browser. Data from serverside instrumentation will also be inaccurate or incomplete.
```
